aiohttp_security/jwt_identity.py

```diff
@@ -0,0 +1,34 @@
+"""Identity policy for storing info in the jwt token.
+
+"""
+
+from .abc import AbstractIdentityPolicy
+try:
+    import jwt
+except ImportError:
+    jwt = None
+
+
+AUTH_HEADER_NAME = 'Authorization'
+
+
+class JWTIdentityPolicy(AbstractIdentityPolicy):
+    def __init__(self, secret, algorithm=None):
+        if jwt is None:
+            raise RuntimeError("Please install pyjwt")
+        self.secret = secret
+        self.algorithm = 'HS256' if algorithm is None else algorithm
+
+    async def identify(self, request):
+        header_identity = request.headers.get(AUTH_HEADER_NAME)
+        identity = jwt.decode(header_identity,
+                              self.secret,
+                              algorithm=self.algorithm)
+
+        return identity['identity']
+
+    async def remember(self, request, response, identity, **kwargs):
+        pass
```

Comment:
should we add `# pragma: no cover` here also?

Reply:
@jettify Yeah :) it's fun that if mark `remember` with `# pragma: no cover` then flake8 doesn't allow to pass tests :)